### DISCUSSION TITLE
nginx gets in stale mode, help to debug please!

### DISCUSSION THREAD

**Discussion by yuriw** [tteck/Proxmox#2498] [Q&A]
I’m running Nginx as a VM on proxmox.
Almost daily Nginx gets to the state so no hosts gets resolved.
The only way to get to a normal state I found so far - restore from backup 

I need help how to debug and troubleshoot this.
Also wonder if anybody else has seem something similar.

TIA

**Comment by tteck** [comment]
This repository doesn't have a Nginx VM!?
